locale/drawers.he.tr

```diff
@@ -6,7 +6,7 @@ Empty=ריקה
 Drawer Controller=
 Digiline Channel=
 Save=
-@1 (@2% full)=@1 (תפוסה של @1%)
+@1 (@2% full)=@1 (תפוסה של @2%)
 @1 @2 (@3% full)=@1 @2 (תפוסה של @3%)
```

Comment:
```suggestion
@1 @2 (@3% full)=@1 @2 (@תפוסה של 3%)
```

Reply:
@lnjX Your change doesn't match my change:

my way: `2%@`
your way: `@3%`

I'm not sure which way is correct in right-to-left languages

Reply:
Ahh i see, i thought you just missed the line. Can we test this? otherwise i'm also fine with just accepting your version.

Reply:
minetest support for he is broken (https://github.com/minetest/minetest/blob/7b3ed3200325ce913a6a2d884ae1ba1ccb08aad5/src/CMakeLists.txt#L668). I'm not sure I we really can test this

Reply:
@lnjX i18ncheck.py reports your line as wrong
```
Error (./drawers/locale/drawers.he.tr:10): Missing placeholder(s) in translation: '@3' 
```